From 071e166644b5d6cfe8a62248b23af62fe57c186e Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Fri, 28 Aug 2015 15:39:48 +0800
Subject: [PATCH 0626/5242] MLK-11454 ARM: dts: imx7d-12x12-lpddr3-arm2:
 enable USB function

commit  aafcf676f9a02db19e2d8f3bad1790c81197144f from
https://source.codeaurora.org/external/imx/linux-imx.git

It doesn't need to do any changes when using lpsr mode for USB.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index d123524..20a0528 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -919,7 +919,7 @@
 	srp-disable;
 	hnp-disable;
 	adp-disable;
-	status = "disabled";
+	status = "okay";
 };
 
 &usbotg2 {
@@ -927,7 +927,7 @@
 	srp-disable;
 	hnp-disable;
 	adp-disable;
-	status = "disabled";
+	status = "okay";
 };
 
 &usdhc1 {
-- 
1.7.9.5

